test/npcomp-run-mlir/conv_2d_nchw.mlir

```diff
@@ -1,38 +1,65 @@
 // RUN: npcomp-run-mlir %s \
 // RUN:   -invoke conv_2d_nchw \
-// RUN:   -arg-value="dense<[[[[1.2]]]]> : tensor<1x1x1x1xf32>" \
-// RUN:   -arg-value="dense<[[[[3.4]]]]> : tensor<1x1x1x1xf32>" \
+// RUN:   -arg-value="dense<0.0> : tensor<2x1x1x1xf32>" \
+// RUN:   -arg-value="dense<0.0> : tensor<1x1x1x1xf32>" \
 // RUN:   -shared-libs=%npcomp_runtime_shlib 2>&1 \
-// RUN:   | FileCheck %s
+// RUN:   | FileCheck %s --check-prefix=BATCH
 
 // RUN: npcomp-run-mlir %s \
 // RUN:   -invoke conv_2d_nchw \
-// RUN:   -arg-value="dense<[[[[1.2, 3.4], [5.6, 7.8]]]]> : tensor<1x1x2x2xf32>" \
-// RUN:   -arg-value="dense<[[[[3.4]]]]> : tensor<1x1x1x1xf32>" \
+// RUN:   -arg-value="dense<0.0> : tensor<1x2x1x1xf32>" \
+// RUN:   -arg-value="dense<0.0> : tensor<2x2x1x1xf32>" \
 // RUN:   -shared-libs=%npcomp_runtime_shlib 2>&1 \
-// RUN:   | FileCheck %s --check-prefix=TINY_SQUARE
+// RUN:   | FileCheck %s --check-prefix=SAME_CHANNELS
 
 // RUN: npcomp-run-mlir %s \
 // RUN:   -invoke conv_2d_nchw \
+// RUN:   -arg-value="dense<0.0> : tensor<1x2x1x1xf32>" \
+// RUN:   -arg-value="dense<0.0> : tensor<1x2x1x1xf32>" \
+// RUN:   -shared-libs=%npcomp_runtime_shlib 2>&1 \
+// RUN:   | FileCheck %s --check-prefix=DIFFERENT_CHANNELS
+
+// RUN: npcomp-run-mlir %s \
+// RUN:   -invoke conv_2d_nchw \
+// RUN:   -arg-value="dense<0.0> : tensor<1x1x2x2xf32>" \
 // RUN:   -arg-value="dense<0.0> : tensor<1x1x1x1xf32>" \
-// RUN:   -arg-value="dense<0.0> : tensor<1x1x1x1xf32>" \
 // RUN:   -shared-libs=%npcomp_runtime_shlib 2>&1 \
-// RUN:   | FileCheck %s --check-prefix=ZEROS
+// RUN:   | FileCheck %s --check-prefix=TINY_SQUARE
+
+// RUN: npcomp-run-mlir %s \
+// RUN:   -invoke conv_2d_nchw \
+// RUN:   -arg-value="dense<0.0> : tensor<1x1x1024x1024xf32>" \
```

Comment:
can we use sizes smaller than 1024? I'm worried the tests will take too long. I think that 32 is probably big enough to test what we care about.